go/arrow/ipc/ipc.go

```diff
@@ -135,6 +136,9 @@ func WithZstd() Option {
 // parallelization. Default is 0.
```

Comment:
Should update the comment that the default is now 1

Reply:
Thanks, updated.